runtime/trappist/src/xcm_config.rs

```diff
@@ -331,7 +337,7 @@ impl pallet_xcm::Config for Runtime {
 	type ExecuteXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
 	type XcmExecuteFilter = Everything;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
-	type XcmTeleportFilter = Nothing;
+	type XcmTeleportFilter = Everything;
```

Comment:
hmmm i think we should provide `TrustedTeleporters` here instead of everything right?

Reply:
Since this is mostly focused on what could or should try to be teleported out of Trappist I didn't apply any criteria. Should we limit this ?

Reply:
Makes all the sense, will implement 🤝

Reply:
Implemented allowing only Native assets to be teleported out of Trappist.